src/pages/Products/Products.js

```diff
@@ -1,8 +1,92 @@
-import React from 'react';
+/* eslint-disable jsx-a11y/alt-text */
+/* eslint-disable react/jsx-key */
+import React, { useState } from 'react';
+import MiniProduct from '../Products/components/MiniProduct';
+import Modal from './components/Modal';
 import './Products.scss';
 
 const Products = () => {
-  return <div>Products</div>;
+  // fetch('http://10.58.52.225:3000/products/category?sc=3&filter=DESC', {
+  //   method: 'GET',
+  //   headers: {
+  //     'Content-Type': 'application/json;charset=utf-8',
+  //   },
+  //   //query: JSON.stringify({filter:"ASC" }),
+  //   //바디 대신에 query:{"main_"}
+  // })
+  //   .then(response => response.json())
+  //   .then(data => console.log(data));
+  const [innerMenu, setInnerMenu] = useState([]);
+
+  fetch('/data/Choisunyoung/innerMenu.json', {
+    method: 'GET',
+  })
+    .then(res => res.json())
+    .then(data => {
+      setInnerMenu(data);
+    });
+
+  const [price, setPrice] = useState([]);
+
+  fetch('data/Choisunyoung/price.json', {
+    method: 'GET',
+  })
+    .then(res => res.json())
+    .then(data => {
+      setPrice(data);
+    });
+
+  return (
+    <div className="product">
```

Comment:
```suggestion
    <div className="products">
```
컴포넌트의 이름과 동일하게 부모 className을 구현해주세요!